### DISCUSSION TITLE
Websocket support?

### DISCUSSION THREAD

**Discussion by yatesco** [http4s/http4s#5662] [Q&A] [answered]
Hi there - In https://http4s.org/v1.0/integrations/ websocket is marked as a question for Ember, which I understand is the new default.

Is there a plan to add websockets to Ember? Also, is Ember a distinct project because the link takes you back to this project, and searching for Ember is flooded with the JS project ;-).

Thanks!

**Comment by armanbilge** [comment, marked as answer]
Ember server has websockets as of at least a few releases ago on 0.22/0.23/1.0:
- https://github.com/http4s/http4s/pull/4803

Ember client websockets are WIP as part of:
- https://github.com/http4s/http4s/pull/5520

**Reply by yatesco** [reply]
thanks - I see the example in https://github.com/http4s/http4s/blob/c9f54a9bb2fc71c81031aa8ddf090e42bb564bd5/examples/ember/src/main/scala/com/example/http4s/ember/EmberServerSimpleExample.scala#L75
